new/LIT7097SanaSom.xml

```diff
@@ -59,6 +66,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
             <div type="bibliography">
                 <listRelation>
                     <relation name="lawd:hasAttestation" active="LIT7097SanaSom" passive="UppQ1524o"/>
+                    <relation name="saws:contains" active="#LIT7097SanaSom" passive="LIT7102QeneYom"/>
                 </listRelation>
             </div>
             <div type="edition" xml:lang="gez">
```

Comment:
I would add `div `for the parts in the edition, and a `corresp `to the lines corresponding to 7102

Reply:
I'm not sure I understood precisely how you mean, so please check if what I did works